tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
```

Comment:
What's with alle the extra lines (in all files)?

Reply:
fixed